theme/_cms-ui.scss

```diff
@@ -241,6 +241,15 @@ body.cms-ui {
     }
   }
 
+  // remove required from Title field
+  .footer-configuration-widget {
+    div.inline.required.field.text[class*='-title'] {
+      label:after {
+        content: none;
+      }
+    }
+  }
```

Comment:
Ok, questo per ora si può mergiare anche così, ma nel frattempo potresti fare anche una PR qui per togliere il required:
https://github.com/RedTurtle/volto-editablefooter/blob/master/src/widget/FooterConfigurationForm.jsx

In fondo è sbagliato che sia required se non c'è validazione...

Allo stesso modo anche questo file si merita una PR per gestire il caso in cui non sia compilato il title
https://github.com/RedTurtle/volto-editablefooter/blob/master/src/components/FooterColumns.jsx